Translate update for Jun (#5)
TOAST-DOCS/RDS-FOR-POSTGRESQL: en/db-security-group.md

* Because DB security groups act as 'stateful', sessions once connected by DB security rules are allowed even if there are no rules in the opposite direction.
* For example, if the first packet on TCP 5432 destined for the DB instance was passed under the 'inbound TCP PORT 5in432' rule, the packet sent from the DB instance to the TCP 5432 port is not blocked.
* However, packets in the opposite direction are also blocked when the session expires because no packets meet the rule for a certain period of time are received.
* It is more efficient to range DB security rules than to add them one by one. If DB security rules increase, the performance might degrade.
* Traffic that is out of state of the session may be blocked.

DB security groups consist of names, descriptions, and a number of DB security rules and the names of DB security groups have the following restrictions.

> * Name of DB security group has to be unique for each region.

Author: ```suggestion
* Name of DB security group must be unique for each region.
```

> * DB security group names can only contain alphabets between 1 and 100 characters, numbers, and some symbols (-, \_, .), and the first letter can only be an alphabetic character.

Author: ```suggestion
* DB security group names can only contain alphabets between 1 and 100 characters, numbers, and some symbols (-, _, .), and the first letter can only be an alphabetic character.
```
